src/node_options.cc

```diff
@@ -1,6 +1,8 @@
+#include "node_options.h"
+
 #include "env-inl.h"
 #include "node_binding.h"
-#include "node_options-inl.h"
```

Comment:
I think removal of this line is what is causing linkage error in `withoutssl` and `withoutintl` modes?

Reply:
Fixed.